CHANGELOG.md

```diff
@@ -1,24 +1,32 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.2.0 alpha 1] - March 4th, 2020
```

Comment:
Nit: `1.2.0-alpha.1`

Reply:
I actually had that at first but needed to change it due to the lint spell checker! @mjc1283, any suggestions on overriding that for this line?

Reply:
Looks like it's something from `markdown-spellcheck` but I'm not sure how to override it (or if it's worth doing so).

Reply:
Tried to add an entry to `.spelling` which I got to work in a local project.

Reply:
Got it with @mjc1283 and @juancarlostong's help. Just had to push new version to https://github.com/optimizely/mdspell-config (and JC is aiming to improve this)